Okay. So after a review of logs and output:
- Daemon queries RPC for addon information, which is proxied from xbmcaddon(), so it's up to Kodi which is there.
``` 
panic: html/template: pattern matches no files: `C:\Users\XBMC\AppData\Roaming\Kodi\addons\plugin.video.quasar\resources\web\*.html`
``` 
This line is directly showing that ADDON.path is totally wrong. (why - who knows).
So we use detector in addon.py to get real path of an addon, and give it to Daemon.

I can't test since i'm on Windows 7 and Windows Store required 10 to install.

--- a/resources/site-packages/quasar/addon.py
+++ b/resources/site-packages/quasar/addon.py
@@ -1,7 +1,13 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
 import xbmcaddon
 
 ADDON = xbmcaddon.Addon()
 ADDON_ID = ADDON.getAddonInfo("id")
 ADDON_NAME = ADDON.getAddonInfo("name")
-ADDON_PATH = ADDON.getAddonInfo("path").decode('utf-8')
 ADDON_ICON = ADDON.getAddonInfo("icon").decode('utf-8')
+try:
+    ADDON_PATH = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__).decode(sys.getfilesystemencoding(), 'ignore')), "..", "..", ".."))
+except:
+    ADDON_PATH = ADDON.getAddonInfo("path").decode('utf-8')

--- a/resources/site-packages/quasar/rpc.py
+++ b/resources/site-packages/quasar/rpc.py
@@ -136,9 +136,10 @@ class QuasarRPCServer(BaseHandler):
     def GetAddonInfo(self):
         info = {}
         for key in ("author", "changelog", "description", "disclaimer",
-                    "fanart", "icon", "id", "name", "path", "profile", "stars",
+                    "fanart", "icon", "id", "name", "profile", "stars",
                     "summary", "type", "version"):
             info[key] = ADDON.getAddonInfo(key)
+        info['path'] = ADDON_PATH
         return info
 
     def AddonFailure(self, addonId):

